certbot/cli.py

```diff
@@ -392,6 +392,11 @@ def parse_args(self):
                         ("Conflicting values for displayer."
                         " {0} conflicts with dialog_mode").format(arg)
                     )
+        else:
+            # -v should imply --text
+            if (parsed_args.verbose_count > flag_default("verbose_count") and
+                not parsed_args.dialog_mode):
```

Comment:
Checking `not parsed_args.dialog_mode` isn't necessary since we're in the `else` branch of `if parsed_args.dialog_mode`.